docs/admin/eventing/channel-configuration.md

```diff
@@ -0,0 +1,29 @@
+# Configure Channel defaults
+
+Knative Eventing provides a `default-ch-webhook` ConfigMap that contains the configuration settings that govern default Channel creation.
+
+The default `default-ch-webhook` ConfigMap is as follows:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: default-ch-webhook
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+    app.kubernetes.io/version: devel
+    app.kubernetes.io/part-of: knative-eventing
+data:
+  # Configuration for defaulting channels that do not specify CRD implementations.
```

Comment:
IMO this is misleading. This ConfigMap is used by the Channel CRD which under the cover creates a platform-specific channel (e.g Kafka, InMemory, etc..).

Reply:
@lionelvillard you mean the comment inside the example yaml file?

Reply:
yes